site/src/docs/components/select/customisation.mdx

```diff
@@ -0,0 +1,187 @@
+---
+slug: '/components/select/customisation'
+title: 'Select - Customisation'
+---
+
+import { Select } from 'hds-react';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Customisation
+
+### Customisation properties
+
+You can use the `theme` property to customise the component. See all available theme variables in the table below.
+
+| Theme variable                                      | Description                                                               |
+| --------------------------------------------------- | ------------------------------------------------------------------------- |
+| '--assistive-color'                                 | Text color of the assistive text.                                         |
```

Comment:
Don't know if this was done different for a reason, but theme variables in tables are stylised on other pages with two ` instead of ' '

`--assistive-color`